analysis_options.yaml

```
@@ -3,4 +3,5 @@ include: package:flutter_lints/flutter.yaml
 
 linter:
   rules:
-  - require_trailing_commas
\ No newline at end of file
+  - require_trailing_commas
+  - prefer_expression_function_bodies
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)